### DISCUSSION TITLE
Where is Prime React Documnetation Version 8?

### DISCUSSION THREAD

**Discussion by ChenReuven** [primefaces/community#513] [PrimeReact] [answered]
https://v8.primereact.org/ is doesn't work.

we are still working with version 8 in our company.

**Comment by melloware** [comment, marked as answer]
I just sent a note to the PrimeTek team it looks like they removed it.

**Reply by ChenReuven** [reply]
great @melloware 

if someone need it like me here is the direct link (i found it in my archive):

https://www.primefaces.org/primereact-v8/
